src/model.ts

```diff
@@ -302,21 +303,25 @@ class ModelClass implements Model, ChangeItem {
     }
 
     const { type, ...fieldChanges } = fieldArgs;
-    const field = new Field(fieldChanges, MutationMode.Update);
+    const field = new Field(fieldChanges as FieldArgs, MutationMode.Update);
```

Comment:
This was giving a TS error 🤔 . Even though `fieldArgs` is "any".

Reply:
<img width="669" alt="Screenshot 2022-02-23 at 07 35 59" src="https://user-images.githubusercontent.com/8969851/155928038-025d91f6-dcc5-4051-a41b-a5d693278225.png">